web/src/lib/elements/ui/link/link.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { Button } from '../button';
+
+  export let text: string;
+  export let url: string;
+  let active: boolean = false;
+
+  onMount(() => {
+    active = window.location.pathname === url;
+  });
+</script>
+
+<Button
```

Comment:
We shouldn't be using a button for an link, that's bad for ARIA. Instead just use an `<a>` tag